helm-chart/kuberay-operator/templates/multiple_namespaces_role.yaml

```diff
@@ -0,0 +1,240 @@
+# Install Role for namespaces listed in watchNamespace.
+# This should be consistent with `role.yaml`, except for the `kind` field.
+{{- if and .Values.rbacEnable .Values.singleNamespaceInstall }}
+{{- $watchNamespaces := default .Values.watchNamespace (list .Release.Namespace) }}
+{{- range $namespace := $watchNamespaces }}
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  labels: {{ include "kuberay-operator.labels" $ | nindent 4 }}
```

Comment:
In a `range` block, `.` no longer refers to the top level scope, so use `$` to access the top-level variables. 

* https://github.com/helm/helm/issues/3684
* https://stackoverflow.com/questions/56671452/helm-chart-error-cant-evaluate-field-values-in-type-interface